### PR TITLE
Remove mpTelemetry mp60 test repeats

### PR DESCRIPTION
**Pull request by Yas-Franco**
Replaces the MicroProfile 6.0 and 6.1 mpTelemetry 2.0 test repeats with MicroProfile 7.0 (EE10 and EE11)

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

File: dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ public static RepeatTests telemetry10and11Repeats(String serverName) {
     }
 
     public static RepeatTests telemetry20Repeats(String serverName) {
-        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, TelemetryActions.MP60_MPTEL20, TelemetryActions.MP61_MPTEL20);
+        return TelemetryActions.repeat(serverName, TelemetryActions.MP14_MPTEL20, TelemetryActions.MP41_MPTEL20, TelemetryActions.MP50_MPTEL20, MicroProfileActions.MP70_EE10, MicroProfileActions.MP70_EE11);
```
Comment (tevans78) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/internal/suite/FATSuite.java:83]: MP Tel 2.0 should run MP70_EE10 in LITE mode ... i.e. that one should be listed first. However, we can fix that in a follow up PR.

**Review by tevans78 — DISMISSED** [review]

**Comment by Yas-Franco** [comment]
#build
#spawn.fullfat.buckets=io.openliberty.microprofile.telemetry.1.0.internal_fat,io.openliberty.microprofile.telemetry.1.0.internal.container_fat,io.openliberty.microprofile.telemetry.1.0.internal_fat_tck,io.openliberty.microprofile.telemetry.1.1.internal_fat_tck,io.openliberty.microprofile.telemetry.2.0.internal_fat_tck

**Review by tevans78 — APPROVED** [review]

**Comment by Yas-Franco** [comment]
#libby

**Pull request merged**